core/src/scene/sceneLoader.cpp

```diff
@@ -73,21 +73,15 @@ void SceneLoader::loadShaderConfig(Node shaders, Style& style, Scene& scene) {
     if (!shaders) { return; }
 
     if (Node extensionsNode = shaders["extensions"]) {
-        std::vector<std::string> extensions;
-        if (extensionsNode.IsScalar()) {
-            extensions.push_back(extensionsNode.as<std::string>());
-        } else if(extensionsNode.IsSequence()) {
-            extensions.reserve(extensionsNode.size());
+        if(extensionsNode.IsSequence()) {
             for (const auto& extNode : extensionsNode) {
-                extensions.push_back(extNode.as<std::string>());
+                char buffer[1000]; //sufficient space
```

Comment:
More seriously though, there's no reason to use `sprintf` now that `snprintf` is available (http://www.cplusplus.com/reference/cstdio/snprintf/)